pkg/sources/reconciler/common/reconcile.go

```diff
@@ -53,7 +53,7 @@ var knativeServingAnnotations = []string{
 // RBACOwnersLister returns a list of OwnerRefable to be set as a the
 // OwnerReferences metadata attribute of a ServiceAccount.
 type RBACOwnersLister interface {
-	RBACOwners(namespace string) ([]kmeta.OwnerRefable, error)
+	RBACOwners(src v1alpha1.EventSource) ([]kmeta.OwnerRefable, error)
 }
```

Comment:
I had to change that interface's signature and propagate the change to _all_ sources, because in case the source sets `WantsOwnServiceAccount -> true`, we want the list of RBAC owners to contain _just_ the current object.

That's why you'll see a lot of tiny changes under `pkg/sources/reconciler`.